libdevcore/CommonJS.h

```diff
@@ -148,9 +149,7 @@ inline u256 jsToU256(std::string const& _s)
 /// 0 Returns 0 in case of failure
 inline int jsToInt(std::string const& _s)
 {
-    int ret = 0;
-    DEV_IGNORE_EXCEPTIONS(ret = std::stoi(_s, nullptr, 0));
-    return ret;
+    return std::stoi(_s, nullptr, 0);
```

Comment:
why do omit the handling of exception?

Reply:
`jsToInt` is only called by RPC to parse transactionIndex when calling getTransactionBy*AndIndex, if the input param is invalid, this interface will return 0, and return the first transaction weirdly, this is unexpected.